Examples/ObjectiveC/LiveDataExample.m

```diff
@@ -0,0 +1,50 @@
+
+#import "LiveDataExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleLiveData = @"LiveDataExample";
+
+@interface LiveDataExample () <MGLMapViewDelegate>
+
+@property (nonatomic, strong, nullable) NSTimer *timer;
+@property MGLShapeSource *source;
+@end
+
+@implementation LiveDataExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle darkStyleURL]];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
+    mapView.delegate = self;
```

Comment:
Can you add a comment above this line that [matches this one](https://github.com/mapbox/ios-sdk-examples/blob/master/Examples/Swift/DrawingACustomMarkerExample.swift#L21) for consistency across our examples?